### ISSUE TITLE
Allow specifying password per flavor/build

### ISSUE THREAD

**Issue by abbyad** [medic/medic-collect#17]
Currently the password is set as part of the Global Credentials, which means all APKs in a build have the same password. Once #16 is complete individual project builds can be done so APKs will unlikely have the wrong password, but it does still mean that the password would need to be set separately. It would be simpler to set the password per flavor, or set it as a build parameter.

**Comment by abbyad** [comment]
When medic-collect is updated we'd want to update all builds so it would be helpful if each flavor has a password, as opposed to a common password field.

**Comment by abbyad** [comment]
Can we update the [Android build documentation](https://github.com/medic/medic-docs/blob/master/development/releasing.md#android-apps) as part of this issue, including a note about adding the passwords to [Travis medic-collect setting](https://travis-ci.org/medic/medic-collect/settings)?

We can open a separate issue or do this as part of another issue if it is easier to manage.

**Comment by kennsippell** [comment]
https://github.com/medic/medic-docs/pull/117
